recycleIT-frontend/src/Components/Services/ServicesList/ServicesSearch/ServicesSearch.tsx

```diff
@@ -39,21 +57,83 @@ const ServicesSearch = (props: IServicesSearchProps) => {
     })
 
     const services = serviceTypes.map((type, i) => {
-        return <MenuItem key={i} value={type.name}>{type.name}</MenuItem>
+        const maxStringLength = 27;
+        let itemName = type.name;
+
+        if (type.name.length > maxStringLength) {
+            itemName = itemName.slice(0, maxStringLength) + '...'
+        }
+        return <MenuItem key={i} value={type.id}>{itemName}</MenuItem>
     })
 
+    const toggleCheckbox = () => {
+        if (!isCheckboxSelected) {
+            selectCurrentLocation()
+        } else {
+            setLocation('')
+        }
+        setIsCheckboxSelected(prev => !prev)
+        setError(false)
+    }
+
+    const requestFacilities = () => {
+        props.fetchServices(location, facilityType)
+    }
+
+    const selectCurrentLocation = () => {  
+        console.log('location request');
+        // will be moved later
+        const API_KEY = 'db6c0e4cf2mshca157dd2f5fb16ap1d4985jsnd8a3ee0592c8',
```

Comment:
Normally you should never commit API keys to remote, always store them in your .env file. It's somewhat acceptable for trainging projects, but normally that's considered a security breach and you should avoid doing that in all cases.

Just make sure to rewoke this API key when you are done with the project, or better generate a new one and store it in .env file. This one should be disabled from google API side

Reply:
Thank you. We know that the keys should be in .env, however, because .env is not supposed to be pushed to github, we wanted to ask, how should we better store the keys?
Would it be ok if each developer creates .env locally, so that we don't have the actual key in our code?